src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,52 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.12.0
+
+**New features**
+
+* Added a new `subctl uninstall` command that removes all Submariner components and dataplane artifacts, such as IP table rules, routes from
+a cluster.
+* Added a new `subctl unexport` command that stops exporting a previously exported service.
+* Added new `subctl cloud prepare` and `subctl cloud cleanup` commands for the Red Hat OpenStack Platform (RHOS).
+* New Globalnet metrics were added:
+  * Count of global Egress IPs allocated at Cluster scope, namespace scope and for selected pods per CIDR.
```

Comment:
```suggestion
  * Count of global Egress IPs allocated at Cluster scope, namespace scope, and for selected pods per CIDR.
```

I think a serial comma would reduce ambiguity here, to make it clear that "namespace scope and for selected pods per CIDR" are not more closely related than "Cluster scope, namespace scope". I think this is a case of point 5 here: https://en.wikipedia.org/wiki/Serial_comma#Arguments_for_and_against